acceptance/openstack/container/experimental/capsules_test.go

```diff
@@ -0,0 +1,28 @@
+package experimental
+
+import (
+	"testing"
+
+	"github.com/gophercloud/gophercloud/acceptance/clients"
+	"github.com/gophercloud/gophercloud/openstack/container/experimental/capsules"
+	th "github.com/gophercloud/gophercloud/testhelper"
+)
+
+func TestCapsuleGet(t *testing.T) {
+	client, err := clients.NewContainerExperimentalClient()
+	if err != nil {
+		t.Fatalf("Unable to create an container experimental client: %v", err)
+	}
+	th.AssertNoErr(t, err)
+	capsuleUUID := "e6c913bb-b4e4-409d-8b71-3e029f196458"
```

Comment:
This is OK for now, but once `List` is implemented, this hard-coded UUID should be removed.